server/logger.js

```diff
@@ -1,7 +1,18 @@
 const log = require("pino")();
 const expressPino = require("express-pino-logger")({ level: "silent", logger: log });
```

Comment:
Not a change in your PR but i wonder why we configure expressPino when we just set it to `silent` with no option to turn it on.

Reply:
:) no idea LOL